### ISSUE TITLE
RubyCocoa is not reliable, get rid of using it (osx/foundation issues etc)

### ISSUE THREAD

**Issue by dottedmag** [mockko/livereload#26]
RubyCocoa is giving a headache to many. Let's just drop it.

This bug is created to merge several bugreports created after 1.4 release.

**Comment by dottedmag** [comment]
Revert to 1.3 (both plugin and gem) until we fix the problem. There's instruction in open tickets.

**Comment by kouky** [comment]
im having this issue as well under plain old ruby 1.8.7 on mac osx

**Comment by kouky** [comment]
can't seem to use the 1.3 gem instead - can't find the compatible chrome extension.

**Comment by flov** [comment]
same problem as mkoukoullis... I can't find the 1.3. extension.
Anybody out there who can provide us with this?

**Comment by zef** [comment]
I have a copy of 1.3, you can download it [here](http://dl.dropbox.com/u/18955/LiveReload.safariextz).

I'll probably delete that file once the issue is resolved...

**Comment by flov** [comment]
@zef, thanks, do you also have the plugin for Chrome in 1.3?

**Comment by zef** [comment]
No, sorry. Just the Safari extension...

**Comment by harmon** [comment]
Here's the error I get on Mac OSX with the latest livereload (v1.4): git://gist.github.com/654960.git

**Comment by chrispederick** [comment]
Running these commands solved the problem for me:

https://gist.github.com/289868

**Comment by flov** [comment]
A thousand thumbs up!!! I got it working that way. Thanks @chrispederick!

**Comment by ionas** [comment]
Thank you very much chrispederick [ https://github.com/chrispederick ] your gist https://gist.github.com/289868 solved it!

**Comment by ionas** [comment]
Does 1.5 fix the RubyCocoa dependency (e.g. removes it?)

**Comment by ionas** [comment]
At least 1.5 runs well (though I did not undo the above GIST)

**Comment by bitboxer** [comment]
Nope, on 1.5 it still needs RubyCocoa :( . Please remove that dependency.

**Comment by NV** [comment]
[guard-livereload](https://github.com/guard/guard-livereload) does not require RubyCocoa and works well with the latest version of LR extension.

**Comment by ionas** [comment]
Now that one looks promising (Guard one)

**Comment by fbehrens** [comment]
I run into this bug, and found that osx/foundation is part of system ruby, 
and its working fine with my system ruby

$ rvm use system

$ sudo gem install livereload

$ livereload

**Comment by bitboxer** [comment]
But I am using ruby 1.9.2, as everybody should. And after installing that (hopefully you do via rvm ;) ), osx/foundation is not available anymore. I could go into the trouble to install it, but it is totally unnecessary. It can be replaced with other solutions. Just look at other projects how they deal with file system notifications under mac os.

**Comment by frankel** [comment]
In Mac (got this info from https://github.com/mockko/livereload/issues/82#issuecomment-1623245 ), you can download a standalone livereload app to work around this issue. http://www.livereload.com,
